clientv3/concurrency/stm.go

```diff
@@ -23,7 +25,7 @@ import (
 type STM interface {
 	// Get returns the value for a key and inserts the key in the txn's read set.
```

Comment:
Get returns the value for only one key to give the stm backend some flexibility in satisfying the request, so that's why I kept it as "a key".